packages/cozy-harvest-lib/src/components/EditAccountModal.jsx

```diff
@@ -131,7 +129,8 @@ export class EditAccountModal extends Component {
               account={account}
               konnector={konnector}
               initialTrigger={trigger}
-              onSuccess={this.handleAccountEditJobSuccess}
+              onSuccess={this.redirectToAccount}
+              onVaultDismiss={this.redirectToAccount}
```

Comment:
I know this is not the goal of this PR but I'm wondering if we can move all the `keys` related stuffs behind a flag?

Reply:
I think we need to do this one of these days yes. @edas what do you think? You seemed to say it wasn't too high-priority?